### DISCUSSION TITLE
Deadline Render Farm Support

### DISCUSSION THREAD

**Discussion by luojunjie2020** [stassius/StableHoudini#39] [General]
Hi, This is amazing tool. and I would like to run multiple SD tasks simultaneously on a Deadline render farm using DeadlineScheduler in houdini. I have noticed that the nodes in Stable Houdini do not offer any scheduling options. Will it be supported in the future, or is there another way to achieve a similar effect?
Thanks

**Comment by stassius** [comment]
It's a great idea, although I'm not familiar with deadline. I will investigate the possibility.
